skimage/transform/tests/test_warps.py

```diff
@@ -160,7 +161,8 @@ def test_rotate_resize_center():
     ref_x45[6, 0] = 1
     ref_x45[7, 0] = 1
 
-    x45 = rotate(x, 45, resize=True, center=(3, 3), order=0)
+    x45 = rotate(x, 45, resize=True, center=(3, 3), order=0,
+                 mode='reflect')
```

Comment:
Why this addition?

Reply:
Before this PR, with `order=0` our internal `fast_warp` function was called but now `ndi.map_coordinate` is called to keep input dtype. This makes this test fail because of border condition effects :confused: